extensions/stac-eo-spec.md

```diff
@@ -86,12 +91,24 @@ numbers of several popular instruments.
 
 ## Associating assets with bands
 
-Asset definitions that contain band data should reference the band index. Each asset should provide a "eo:bands" property that is an array of 0 based indexes to the correct Band Objects.
+Asset definitions that contain band data should reference the band index and any additional asset specific band data.
 
 ### Item `Asset Object` fields
-| Field Name | Type     | Description                                  |
-| ---------- | -------- | -------------------------------------------- |
-| eo:bands   | [number] | Lists the band names available in the asset. |
+
+| Field Name | Type                | Description                                                             |
+| ---------- | ------------------- | ----------------------------------------------------------------------- |
+| eo:bands   | [Asset Band Object] | Lists the bands available in an asset and any asset specific band data. |
+
+### Asset Band Object
+
+| Field Name | Type   | Description                                                  |
+| ---------- | ------ | ------------------------------------------------------------ |
+| index      | number | The band index as specified in the Band Object                |
+| unit       | string | Unit of measurements, preferably following the singular unit names in the [UDUNITS2 database](https://ncics.org/portfolio/other-resources/udunits2/). |
+| offset     | number | Offset to convert band values to the actual measurement scale. Defaults to `0`. |
+| scale      | number | Scale to convert band values to the actual measurement scale. Defaults to `1`. |
```

Comment:
Is the asset band object just a duplicate of the band object? I thought we were just specifying a lookup number here and we would define the metadata in the Band Object?